### PR TITLE
Log progress for arrow construct calls

### PR DESCRIPTION
**Pull request by DarthMax**
* Remove multi threading
* Manually chunk data frames
* Show progress bar for node and relationship import

### CODE REVIEW

**Review by adamnsch — COMMENTED** [review]
Very cool stuff :) Should we also have an "Improvements" changelog entry?

**Comment by FlorentinD** [comment]
@DarthMax lets get this cool stuff in to main :)

**Comment by adamnsch** [comment]
Actually, when I use this locally I get progress bars that finish at 33%. Probably not what we want

**Comment by DarthMax** [comment]
> Actually, when I use this locally I get progress bars that finish at 33%. Probably not what we want

Do you have some sample code that causes this?

**Comment by adamnsch** [comment]
> > Actually, when I use this locally I get progress bars that finish at 33%. Probably not what we want
> 
> Do you have some sample code that causes this?

Yes :)

1. Check out https://github.com/neo4j/graph-data-science-client/pull/226
2. pip install ogb
3. `pytest graphdatascience/tests/integration/test_graph_ops.py -s -k ogbn_arxiv --include-enterprise`

**Comment by DarthMax** [comment]
> > > Actually, when I use this locally I get progress bars that finish at 33%. Probably not what we want
> > 
> > 
> > Do you have some sample code that causes this?
> 
> Yes :)
> 
>     1. Check out [WIP: Add convenience loading of OGB graphs #226](https://github.com/neo4j/graph-data-science-client/pull/226)
> 
>     2. pip install ogb
> 
>     3. `pytest graphdatascience/tests/integration/test_graph_ops.py -s -k ogbn_arxiv --include-enterprise`

I pushed a fix. I used the wrong method to calculate the number of rows. Pro Tip it is not `size` :see_no_evil:

**Comment by adamnsch** [comment]
> > > > Actually, when I use this locally I get progress bars that finish at 33%. Probably not what we want
> > > 
> > > 
> > > Do you have some sample code that causes this?
> > 
> > 
> > Yes :)
> > ```
> > 1. Check out [WIP: Add convenience loading of OGB graphs #226](https://github.com/neo4j/graph-data-science-client/pull/226)
> > 
> > 2. pip install ogb
> > 
> > 3. `pytest graphdatascience/tests/integration/test_graph_ops.py -s -k ogbn_arxiv --include-enterprise`
> > ```
> 
> I pushed a fix. I used the wrong method to calculate the number of rows. Pro Tip it is not `size` see_no_evil

awesome! :clap:

File: graphdatascience/query_runner/arrow_graph_constructor.py [hunk 1]
```diff
@@ -53,33 +62,36 @@ def _partition_dfs(self, dfs: List[DataFrame]) -> List[DataFrame]:
...
         partitioned_dfs = self._partition_dfs(dfs)
 
         with ThreadPoolExecutor(self._concurrency) as executor:
-            futures = [executor.submit(self._send_df, df, entity_type) for df in partitioned_dfs]
-
-            wait(futures)
+            futures = [executor.submit(self._send_df, df, entity_type, pbar) for df in partitioned_dfs]
 
-            for future in futures:
+            for future in concurrent.futures.as_completed(futures):
```
Comment (adamnsch) [inline graphdatascience/query_runner/arrow_graph_constructor.py:94]: Cool!

**Review by adamnsch — APPROVED** [review]
Looks good! But we should have a changelog entry ;)

**Pull request merged**